test/functional/rpc_blockchain.py

```
@@ -417,10 +417,58 @@ def _test_getdifficulty(self):
 
     def _test_getnetworkhashps(self):
         self.log.info("Test getnetworkhashps")
-        hashes_per_second = self.nodes[0].getnetworkhashps()
+        assert_raises_rpc_error(
+            -3,
+            textwrap.dedent("""
```

Ruff (0.12.2)

Undefined name `textwrap`

(F821)